src/DocumentDB.ChangeFeedProcessor/LeaseManagement/DocumentServiceLease.cs

```diff
@@ -72,6 +72,15 @@ public DateTime Timestamp
         [JsonProperty("_ts")]
         private long TS { get; set; }
 
+        [JsonProperty("LeaseToken", NullValueHandling = NullValueHandling.Ignore)]
```

Comment:
Can we add a comment about this is for migration from V3?

Reply:
Done